<launch>
  <arg name="com_mode" default="auto_update" /><!-- or auto_update, polling -->
  <arg name="remap" default="false" /><!-- Remap joint_states topic. e.g. for joint_state_publisher -->
  <arg name="force" default="40" />

  <remap from="/joint_states" to="/joint_states_gripper" if="$(arg remap)"/>

  <node  name="wsg50_gripper"  pkg="wsg_50_driver" type="wsg_50_ip" output="screen" ns="iiwa">
	  <param name="ip" type="string" value="172.31.1.160"/>
	  <param name="port" type="int" value="1000"/>
    <param name="protocol" type="string" value="tcp"/>
    <param name="com_mode" type="string" value="$(arg com_mode)"/>
    <param name="rate" type="double" value="50"/> <!-- WSG50 HW revision 2: up to 30 Hz with script; 140Hz with auto_update -->
    <param name="grasping_force" type="double" value="$(arg force)"/>
  </node>

</launch>

